airbyte-config/config-models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -58,3 +58,10 @@ properties:
   protocolVersion:
     type: string
     description: the Airbyte Protocol version supported by the connector
+  normalizationRepository:
+    type: string
+  normalizationTag:
+    type: string
+  supportsDbt:
```

Comment:
Do we foresee any other transformation libraries being supported in the future?  Wondering if instead of this being a boolean, this should instead be an enum with `dbt` as one of the values.

Reply:
To be honest, I'm not aware of any plans to add more transform libraries. Maybe @evantahler knows better about such plans? But, if a new transformation library will appear in the future, it will be possible to switch from boolean to using enam in the database quite quickly.

Reply:
I know of no other transform libraries other than dbt... for now anyway.  I've heard talk that perhaps in the future we want to allow folks to provide SQL files, but that's not on the roadmap yet.